content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -111,25 +113,58 @@ to understand how `X-Forwarded-For` headers and trusted client addresses are det
 1. Deploy `httpbin` in the `httpbin` namespace:
 
     {{< text syntax=bash snip_id=apply_httpbin >}}
-    $ kubectl apply -n httpbin -f samples/httpbin/httpbin.yaml
+    $ kubectl apply -n httpbin -f @samples/httpbin/httpbin.yaml@
     {{< /text >}}
 
 1. Deploy a gateway associated with `httpbin`:
 
-    {{< text syntax=bash snip_id=deploy_httpbin_gateway >}}
-    $ kubectl apply -n httpbin -f samples/httpbin/httpbin-gateway.yaml
-    {{< /text >}}
+{{< tabset category-name="config-api" >}}
+
+{{< tab name="Istio classic" category-value="istio-classic" >}}
+
+{{< text syntax=bash snip_id=deploy_httpbin_gateway >}}
+$ kubectl apply -n httpbin -f @samples/httpbin/httpbin-gateway.yaml@
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+{{< text syntax=bash snip_id=deploy_httpbin_k8s_gateway >}}
+$ kubectl apply -n httpbin -f @samples/httpbin/gateway-api/httpbin-gateway.yaml@
+$ kubectl wait --for=condition=ready gtw -n httpbin httpbin-gateway
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
 
 1. Set a local `GATEWAY_URL` environmental variable based on your Istio ingress gateway's IP address:
```

Comment:
Looking at the Netlify output, the renumbering started with number 1. Was that a tabset issue?

Reply:
Fixed. Good catch!